Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3075 @@
+{
```

Comment:
<ul><li>statsmodels is not used anymore in the NB</li><li>seaborn should be placed after pymc3</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>